README.md

```diff
@@ -1,33 +1,25 @@
 # How to contribute
 
 Before you contribute start with a bit of research. Check if there are any rules for what you're about to add. 
-You can see the entire rules set under `/rules/<engine>/<language>/`, If no rule is found, let's get started. 
+You can see the entire rules set under `/rules/<engine>/<language>/<framework>/`, If no rule is found, let's get started. 
 
-### 1. Select a rule name and create your contribution folder
-In the "rules" folder, under the engine & langauge you want to write the rules for, create a folder with the name of your new rule.
+`[Important]` Do not plagiarise other sources. While it's useful to read blogs and look at what others are doing, plagiarism will not be accepted. 
 
-e.g. if you want to create a rule to detect `SQL Injection` in the `NodeJS` framework using the `semgrep` engine, your rule name may be called: `NodeJS_SQL_Injection` and the created folder will be `/rules/semgrep/JavaScript/NodeJS_SQL_Injection`
+### 1. Select a rule name and its folder
+Our rules repo is set to hold rules to multiple different open-sourced engines. As such, the hierarchy we use to store the rules starts on the engine, followed by langauge, and framework (when applicable) each rule should be located in the folder of the framework it is written for. 
+Make sure the name you pick for the rule is as descriptive as possible, this will make it easier for others to find that such rule already exists.
```

Comment:
```suggestion
Make sure the name you pick for the rule is as descriptive as possible, this will make it easier for others to find that such a rule already exists.
```